crates/payload/basic/src/lib.rs

```diff
@@ -967,6 +990,49 @@ fn commit_withdrawals<DB: Database<Error = Error>>(
     })
 }
 
+/// TODO: docs
+fn pre_block_contract_call<DB>(
```

Comment:
this will be required for any kind of payload building so I think we should move this to the payload/builder crate

Reply:
Let's track in new good-first-issue